probes/ntp_amp/generate_report.py

```diff
@@ -2,17 +2,16 @@
 
 import re
 import sys
-from lxml import etree
-
-ntp_amplification_test = etree.Element('NTP_AMPLIFICATION_TEST')
 
 stdout_data = sys.stdin.readlines()
 regex = (re.search('remote\saddress\s*port\slocal\saddress\s*count\sm\sver\srstr\savgint\s*lstint', stdout_data[0])) if stdout_data else ""
-if regex:
-    ntp_amplification_test.text = "Monlist response is enabled"
-    print("ERROR")
+if regex: 
+    print('ERROR')
+    print('NTP port on UDP is open')
```

Comment:
We rather report: "The machine exposes NTP configuration that can be abused for amplification attacks"